lib/rails/generators/intercom/config/intercom.rb.erb

```diff
@@ -3,6 +3,10 @@ IntercomRails.config do |config|
   #
   config.app_id = ENV["INTERCOM_APP_ID"] || "<%= @app_id %>"
 
+  # == Intercom session_duration
+  #
+  config.session_duration = <%= @session_duration %>
```

Comment:
we want it to be a length of time rather than a specific timestamp, that way it will set the cookie expiry dynamically when the client initialises